exercises/practice/leap/leap_test.sql

```diff
@@ -0,0 +1,59 @@
+-- Create database:
+.read ./create_fixture.sql
+
+-- Read user student solution and save any output as markdown in user_output.md:
+.mode markdown
+.output user_output.md
+.read ./success.sql
+.output
+
+-- Create a clean testing environment:
+DROP TABLE IF EXISTS main.tests;
+CREATE TABLE IF NOT EXISTS main.tests (
+    -- uuid and name are taken from the test.toml file
+    uuid TEXT PRIMARY KEY,
+	name TEXT NOT NULL,
+    -- The following section is needed by the online test-runner
+    status TEXT DEFAULT "fail",
+    message TEXT,
+    output TEXT,
+    test_code TEXT,
+    task_id INTEGER DEFAULT NULL,
+    -- Here are columns for the actual tests
+    year INT NOT NULL,
+    result BOOL NOT NULL
+);
+
+INSERT INTO tests (name, uuid, year, result)
+    VALUES
+        -- Every test case from the .meta/tests.toml file gets its own row:
+        ("year not divisible by 4 in common year", "6466b30d-519c-438e-935d-388224ab5223", 2015, 0),
```

Comment:
My version of sqlite3 doesn't like `"` and requires `'`. I don't know how uniform that need is.

Reply:
When we set up the cli tests, we should try to check this behavior.
I have tested the double quotes locally with version 3.37.2 on ubuntu 22.04.
The test runner is fine with it on alpine 3.18 with sqlite3 version 3.41.2.

Reply:
But we should use single quotes:

> Use single-quotes, not double-quotes, around string literals in SQL. This is what the SQL standard requires. Your WHERE clause expression should read: column1='column1'

> SQL uses double-quotes around identifiers (column or table names) that contains special characters or which are keywords. So double-quotes are a way of escaping identifier names. Hence, when you say column1="column1" that is equivalent to column1=column1 which is obviously always true.

From https://www.sqlite.org/faq.html Item 24.